angular/src/app/modules/retros/retro/retro-detail/retro-detail.component.ts

```diff
@@ -287,6 +288,12 @@ export class RetroDetailComponent
     this.showDialog(retroDetail, ActionDialog.EDIT);
   }
 
+  addMulti(): void {
+    let item = {} as RetroDetailCreateEditDto;
+    item.retroId = this.retroId;
+    this.showDialogMulti(item);
```

Comment:
change three rows above to single row
`this.showDialogMulti({retroId: this.retroId});`